packages/core/core/src/Strapi.ts

```diff
@@ -32,7 +32,8 @@ import type {
 
 import loadConfiguration from './core/app-configuration';
 
-import * as factories from './factories';
+import * as factories from './factories/factories';
```

Comment:
We are kind of breaking encapsulation if we need to import the factories module and also a file inside it. (consider the factories folder is a single module, it's public API should be in the index and that's it) 

From this it means easer the symbols should be public or we should export some utils to check if a controller is customize (my reco) as part of the public API for that module

Reply:
Make sense :thinking:

Reply:
Done in [f9cc11c](https://github.com/strapi/strapi/pull/19061/commits/f9cc11c307a2d2756255075a74efa16e096e3fb9)